vignettes/articles/00-prerequisites.Rmd

```diff
@@ -22,17 +21,15 @@ This vignette describes the necessary prerequisites for installing `r2dii.climat
 
 2DII climate transition stress testing analysis is performed using R packages. R is a programming language in which the source code is written. To use R, it is easiest to use an Integrated Development Environment (IDE), in this case RStudio. It is recommended for the person performing the analysis to have basic coding skills.
 
-Programming language:
+**Programming language**
 
-R - version must be 3.4 or higher (it is best practice to download the most recent version of R). This can be downloaded for Windows from https://cran.studio.com/bin/windows/base
+* R - version must be 3.4 or higher (it is best practice to download the most recent version of R). This can be downloaded for Windows from https://cran.studio.com/bin/windows/base
+* In most cases, this will be downloadable from the bank's internal software repository server subject to approval by the bank's IT department.
```

Comment:
should we already make this a bit more generic?
"In most cases, this will be downloadable from the financial institution's internal software repository..."